enterprise/metrics/src/main/java/org/neo4j/metrics/output/GangliaOutput.java

```diff
@@ -56,42 +59,48 @@ public GangliaOutput( Config config, MetricRegistry registry, Log logger, String
     @Override
     public void init() throws IOException
     {
-        if ( config.get( gangliaEnabled ) )
-        {
-            // Setup Ganglia reporting
-            hostnamePort = config.get( gangliaServer );
-            final GMetric ganglia = new GMetric(
-                    hostnamePort.getHost(),
-                    hostnamePort.getPort(),
-                    GMetric.UDPAddressingMode.MULTICAST,
-                    1 );
-
-            gangliaReporter = GangliaReporter.forRegistry( registry )
-                                             .prefixedWith( prefix )
-                                             .convertRatesTo( TimeUnit.SECONDS )
-                                             .convertDurationsTo( TimeUnit.MILLISECONDS )
-                                             .filter( MetricFilter.ALL )
-                                             .build( ganglia );
-        }
+        // Setup Ganglia reporting
+        final GMetric ganglia = new GMetric( hostnamePort.getHost(), hostnamePort.getPort(), MULTICAST, 1 );
```

Comment:
will just leave it here for now to not forget, have nothing to do with this commit. Why it's multicast?